client/landing/stepper/declarative-flow/internals/steps-repository/start-writing-done/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { Button, ConfettiAnimation } from '@automattic/components';
+import { StepContainer } from '@automattic/onboarding';
+import { useTranslate } from 'i18n-calypso';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { useSiteSlugParam } from 'calypso/landing/stepper/hooks/use-site-slug-param';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import StartWritingDoneSitePreview from './site-preview';
+import type { Step } from '../../types';
+
+import './styles.scss';
+
+const StartWritingDone: Step = () => {
+	const translate = useTranslate();
+	const siteSlug = useSiteSlugParam();
+	return (
+		<StepContainer
+			stepName="start-writing-done"
+			isWideLayout={ true }
+			hideBack={ true }
+			flowName="free"
+			formattedHeader={
+				<FormattedHeader
+					id="start-writing-done-header"
+					headerText={ translate( 'Your blog’s ready!' ) }
+					subHeaderText={ translate( 'Now it’s time to connect your social accounts.' ) }
+					align="center"
+					subHeaderAlign="center"
+				/>
+			}
+			stepContent={
+				<>
+					<ConfettiAnimation />
+					<div className="start-writing-done__top-content">
+						<div className="start-writing-done__top-content-main">
+							<div className="start-writing-done__top-content-title">Notes by Livro</div>
+							<div className="start-writing-done__top-content-description">{ siteSlug }</div>
```

Comment:
Question: if a user purchases the domain, will the siteSlug in the params be updated to the domain that the user chose?

Reply:
Good question, thanks to you I found the same issue in the sidebar. [Fixed here](https://github.com/Automattic/wp-calypso/pull/76584/commits/4fa0d58f4cd5ef7332d12b939321a71d86dc9bf0).